src/handlers/musicHandler.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Responsible for creating and managing musicplayers per guild
+ */
+
+import { Collection } from 'discord.js';
+import MusicPlayer from '../models/MusicPlayer';
+import DiscordClient from '../models/client';
+
+export default class MusicHandler extends Collection<string, MusicPlayer> {
+    readonly client: DiscordClient;
+
+    constructor(client: DiscordClient) {
+        super();
+        this.client = client;
+    }
+
+    getOrCreate = (guildId: string): MusicPlayer =>
```

Comment:
i guess it's just stylistic
i learned to heavily use arrow functions, maybe because i'm react brained
could you elaborate on why arrow fns are less readable?